packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -226,3 +222,13 @@ module.exports = {
     tls: 'empty'
   }
 };
+
+// @remove-on-eject-begin
+// Override babel-loader options to not read .babelrc.
+var babelLoader = module.exports.module.loaders.find(l => l.loader === 'babel');
+babelLoader.query = babelLoader.query || {};
+babelLoader.query.babelrc = false;
+// We embed the preset instead of referring to it because otherwise
+// `env` options in the preset do not get taken into account.
```

Comment:
Presets can use `env`. I think it’s our peculiar way of using them (`babelrc: false`) that might be breaking this. It would be amazing if you could investigate why this happens but I want to ship this today so going to leave it as is for now.